### PR TITLE
fix: width of text in video element

### PR DESCRIPTION
**Pull request by rlaco-qualityunit**
**Changes proposed in this Pull Request**
Changed video position in section on desktop. Width of text was fixed.

**Testing instructions**
Please check if my solution works as was requested

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#[502]

### CODE REVIEW

File: assets/styles/components/_BlockVideo.scss [hunk 1]
```diff
@@ -196,7 +196,7 @@ $positions: ("left", "right");
 				@media (min-width: 800px) {
 
 					.wrapper {
-						padding: 0;
+						//padding: 0;
```
Comment (jremen) [inline assets/styles/components/_BlockVideo.scss:199]: Remove commented code

File: assets/styles/components/_BlockVideo.scss [hunk 2]
```diff
@@ -265,11 +267,15 @@ $positions: ("left", "right");
 				@media (min-width: $breakpoint-desktop-box) {
 
 					.wrapper {
-						#{$pos}: calc(-50vw + 690px - 20px);
+						//#{$pos}: calc(-50vw + 690px - 20px);
```
Comment (jremen) [inline assets/styles/components/_BlockVideo.scss:270]: Are we sure we want to remove dynamic calculation? If so, remove commented out code. Never leave such code undeleted.

**Review by jremen — CHANGES_REQUESTED** [review]
Read the comments in code

**Review by jremen — APPROVED** [review]

**Comment by rlaco-qualityunit** [comment]
due to commit errors, new branch "502_video_element_text_fix" was created

**Pull request closed without merging**